fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/election/ElectionQuestion.kt

```diff
@@ -2,22 +2,32 @@ package com.github.dedis.popstellar.model.network.method.message.data.election
 
 import com.github.dedis.popstellar.model.Immutable
 import com.github.dedis.popstellar.model.objects.Election
+import com.github.dedis.popstellar.utility.MessageValidator.verify
 import com.google.gson.annotations.SerializedName
 import java.util.Collections
 import java.util.Objects
 
 @Immutable
 /** Constructor for a data Question, for the election setup */
 class ElectionQuestion(electionId: String, question: Question) {
-  val question: String = question.title
-  val id: String = Election.generateElectionQuestionId(electionId, question.title)
+  val question: String
+  val id: String
 
-  @SerializedName(value = "voting_method") val votingMethod: String = question.votingMethod
+  @SerializedName(value = "voting_method") val votingMethod: String
 
-  @SerializedName(value = "ballot_options")
-  val ballotOptions: List<String> = Collections.unmodifiableList(question.ballotOptions)
+  @SerializedName(value = "ballot_options") val ballotOptions: List<String>
 
-  @SerializedName(value = "write_in") val writeIn: Boolean = question.writeIn
+  @SerializedName(value = "write_in") val writeIn: Boolean
+
+  init {
+    verify().isNotEmptyBase64(electionId, "election ID").validQuestions(listOf(question))
```

Comment:
Saw that you did a commit named `ElectionQuestion : removed overlapping Question validations` but it's actually an empty commit (just some formatting), so I guess you forgot to remove the validQuestions check here

Reply:
I actually removed the redundant checks performed in `validQuestions` (calling `validQuestion` for each Question in the list). validQuestions is still used, as it checks for duplicates and requires the list not to be empty. Maybe it makes it not relevant anymore to be a function in `MessageValidator` tho.

Reply:
Yeah the point was: why are you still calling `.validQuestions(listOf(question))` in `ElectionQuestion`? The fact that the list is not empty and without duplicates is given by the construction of it (a list of 1 element)

Reply:
oh yes that makes a lot of sense, I overlooked